src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const employeesList = document.querySelector('ul');
+const employees = [...document.querySelectorAll('li')];
+
+const salaryToNum = (salary) => Number(salary.replace(',', '').slice(1));
```

Comment:
We can use more clear name to this function.
For example:
`convertSalaryToNumber` or `parseSalary`.
Also we can use `parseInt` for this case(Up to u)